data/quests.ts

```diff
@@ -1,6 +1,109 @@
 import { ListType } from "./types";
 
 export const Quests: Array<ListType> = [
+  /* TEMPLATE     
+  {
+    // Can be left blank to be filled later
+    id: "",
+
+    // Name of the NPC/ending
+    name: "",
+
+    requirements: [
+      {
+        // Can be left blank to be filled later
+        id: "",
+
+        // Short sentence in brief describing necessary actions with few details
+        // Acceptable details:
+        // - Nearby Site of Grace, or Map Section
+        // - Necessary bosses and items
+        // - Dialog options
+        description: ""
+      },
+    ]
+  },
+  */
+  {
+    id: "",
+    name: "Kenneth Haight (Incomplete)",
+    requirements: [
+      {
+        id: "",
+        description: "Talk to Kenneth in West Limgrave near Third Church of Marika site of grace.",
+      },
+      {
+        id: "",
+        description: "Clear out enemies in Fort Haight in West Limgrave.",
+      },
+      {
+        id: "",
+        description: "Talk to Kenneth at his first location. Receive a reward.",
+      },
+      {
+        id: "",
+        description: "Talk to Kenneth at Fort Haight.",
+      },
+      {
+        id: "",
+        description: "To Be Determined ...",
+      },
+    ],
+  },
+  {
+    id: "",
+    name: "Nepheli Loux (Incomplete)",
+    requirements: [
+      {
+        id: "",
+        description: "Meet Nepheli Loux in Stormveil Castle, nearby the Secluded Corridoor site of grace.",
+      },
+      {
+        id: "",
+        description: "Defeat Godrick the Grafted.",
+      },
+      {
+        id: "",
+        description: "Visit Roundtable Hold and talk to Nepheli. Receive the Arsenal Talisman.",
+      },
+      {
+        id: "",
+        description: "(Optional) Talk to Seluvis and receive a Potion for Nepheli. Requires Ranni's quest.",
+      },
+      {
+        id: "",
+        description: "Meet Nepheli at the Village of the Albinaurics.",
+      },
+      {
+        id: "",
+        description: "Defeat Omenkiller at the Village.",
+      },
+      {
+        id: "",
+        description: "Visit Roundtable Hold and talk to Nepheli.",
+      },
+      {
+        id: "",
+        description: "Visit the Four Belfries in West Liurnia, and teleport to the Precipice of Anticipation (Requires 1 Imbued Stonesword Key).",
+      },
+      {
+        id: "",
+        description: "Defeat the Grafted Scion.",
+      },
+      {
+        id: "",
+        description: "Explore the Church of Anticipation, and find Stormhawk Ashes.",
+      },
+      {
+        id: "",
+        description: "(Choice) Give Nepheli the Potion from Seluvis or the Ashes (at Roundtable Hold).",
+      },
+      {
+        id: "",
+        description: "To Be Determined ...",
```

Comment:
I'd say it's worth to include what is known about this quest line after patch 1.03. 

```suggestion
        id: "",
        description: "Talk to Nepheli at the Stormveil Castle throne room.",
```